pyaccel/optics/driving_terms.py

```diff
@@ -0,0 +1,257 @@
+"""Calculate Resonance Driving Terms."""
+
+import matplotlib.pyplot as _mplt
+import numpy as _np
+
+from .. import lattice as _lattice
+from ..graphics import draw_lattice as _draw_lattice
+from .twiss import calc_twiss as _calc_twiss
+
+
+class FirstOrderDrivingTerms:
+    """Calculate first order resonance driving terms."""
```

Comment:
@fernandohds564 , s it possible to add a ref link to a simple didactic explanations of the RDTs ?

Reply:
Sure @xresende. I added a link to Bengtsson's report in the description.